dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESIndexAPI.java

```diff
@@ -612,10 +612,12 @@ public synchronized CreateIndexResponse createIndex(final String indexName, Stri
 
 		map.put("number_of_shards", shards);
 		map.put("index.auto_expand_replicas", "0-all");
-		map.put("index.mapping.total_fields.limit",
-			Config.getIntProperty("ES_INDEX_MAPPING_TOTAL_FIELD_LIMITS", 5000));
-        map.put("index.mapping.nested_fields.limit",
-                Config.getIntProperty("ES_INDEX_MAPPING_NESTED_FIELDS_LIMITS", 5000));
+		if (!map.containsKey("index.mapping.total_fields.limit")) {
```

Comment:
We need to remove our non-standard way of configuring elasticsearch and instead rely on these changes:

https://github.com/dotCMS/core/blob/e4e22c1deb5bf13abbab9496c59680da1aeafe6e/dotCMS/src/main/resources/es-content-settings.json#L29-L33

Reply:
Per my conversation with Jose, this looks good. I just want to be sure that our `es-content-settings.json` is correct and should be the place where we make these changes in the future.

Reply:
Discussed this comment with @wezell , and confirmed that we're reading config files from the `es-content-settings.json` file only, and not using the `dotcms-config-cluster.properties` file anymore. The IF in there is just meant to set default values in case the JSON file is not specifying them.